script/elevate-cpanel.PL

```diff
@@ -954,6 +954,7 @@ Restore removed packages during the previous stage.
 
 sub run_stage_4 ($self) {
     Cpanel::OS::flush_disk_caches();
```

Comment:
wonder if `flush_disk_caches` should not imply `clear_cache` also 
but this LGTM

Reply:
That would probably complicate cPanel tests